proposals/lip-0052.md

```diff
@@ -2288,51 +2239,47 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
 
 - Initial checks on the properties of `genesisBlockAssetObject`:
 
-  - Across all elements of the `NFTSubstore` array, all values given for `nftID` must be unique.
-  - For all elements of the `NFTSubstore` array, values given for `owner` must have either length `LENGTH_ADDRESS` bytes (representing a user address) or `LENGTH_CHAIN_ID` bytes (representing a chain ID).
-  - For every element in the `NFTSubstore` array, there should exist unique entry having the same value for `nftID` property in either:
-    - The `userSubstore` array, if the value of the `owner` property has length `LENGTH_ADDRESS` bytes, or
-    - The `escrowSubstore` array, if the value of the `owner` property has length `LENGTH_CHAIN_ID` bytes.
-  - The `NFTSubstore` must be in lexicographical order of `nftID`.
-    - The `attributesArray` must be in lexicographic order of `module` name.
-  - Across all elements of the `userSubstore` array, the pairs `address`, `nftID` must be unique: each pair appears at most once, although a given `address` can appear multiple times with different unique `nftID`. Furthermore, each `nftID` should also be unique.
-  - The `userSubstore` array must be in lexicographical order of `address`. For a given `address`, the entries must be in lexicographic order of `nftID`.
-  - Across all elements of the `escrowSubstore` array, the pairs `escrowedChainID`, `nftID` must be unique: each pair appears at most once, although a given `escrowedChainID` can appear multiple times with different unique `nftID`. Furthermore, each `nftID` should also be unique.
-  - For a given element of either the `userSubstore` or `escrowSubstore` arrays, there should exist a unique element in the `NFTSubstore` array, having the same value of the `nftID`.
+  - The `NFTSubstore` must be in lexicographical order of `nftID`. Across all elements of the `NFTSubstore` array:
+    - All values given for `nftID` must be unique.
+    - All values given for `owner` must have either length `LENGTH_ADDRESS` bytes (representing a user address) or `LENGTH_CHAIN_ID` bytes (representing a chain ID).
+    - The `attributesArray` must be in lexicographic order of `module` name, and all values of `module` must be unique.
   - The `supportedNFTsSubstore` array, if non-empty, must adhere to one of the following two mutually exclusive conditions:
     - Have a single entry with `chainID` of length `0` and no other entries. This indicates that all NFTs are supported, i.e., `chainID == ALL_SUPPORTED_NFTS_KEY`. In this case, the `supportedCollectionIDArray` should be an empty array.
     - Across all elements, the `chainID` must be unique and have length `LENGTH_CHAIN_ID`. Furthermore, the `supportedNFTsSubstore` array must be in lexicographic order of `chainID`. Also, for each entry of this array, the `supportedCollectionIDArray` should be in lexicographic order.
```

Comment:
```suggestion
    - Otherwise, across all elements, the `chainID` must be unique and have length `LENGTH_CHAIN_ID`. Furthermore, the `supportedNFTsSubstore` array must be in lexicographic order of `chainID`. Also, for each entry of this array, the `supportedCollectionIDArray` should be in lexicographic order.
```